apps/7_streamlit_chat_rag/README.md

```diff
@@ -70,9 +70,22 @@ To see how the RAG feature works, watch the video below.
 
 [![Streamlit Chat App with RAG Feature](https://img.youtube.com/vi/ummiu-rzYvs/0.jpg)](https://youtu.be/ummiu-rzYvs)
 
-### How to customize
+## How to customize
 
-You can customize the chat app by modifying the following codes:
+### Customize your application purpose
 
-- [main.py](./main.py): `CUSTOM_SYSTEM_PROMPT` variable which defines the system prompt
-- [tools/fetch_contents.py](./tools/fetch_contents.py): `fetch_contents` function comments which is passed to the LLM model
+The application purpose is defined in a system prompt. By default, the application provides information about the fictional company Contoso Corporation.
+
+To change the behavior of the application, modify the `CUSTOM_SYSTEM_PROMPT` variable to fit your needs.
+Also designing prompts, so called "prompt engineering", is important to get the best results from the model. There are many resources available to help you design optimal prompts. Here are some recommendations:
+
+- [Prompt Engineering Guide](https://www.promptingguide.ai/)
+<!-- FIXME: Add prompt engineering sources, slides -->
```

Comment:
@dahatake -san
I'd like to refer to your repos, docs or slides link :)

Reply:
fixed by [0d46d1f](https://github.com/ks6088ts-labs/workshop-azure-openai/pull/101/commits/0d46d1fdcda9cec018ee99f4951df4ceba76c4a6)